consensus/src/consensusdb/mod.rs

```diff
@@ -187,4 +191,34 @@ impl ConsensusDB {
         iter.seek_to_first();
         Ok(iter.collect::<Result<HashMap<HashValue, QuorumCert>>>()?)
     }
+
+    pub fn save_node(&self, node: &Node) -> Result<(), DbError> {
```

Comment:
No get and delete methods?

Reply:
this part is not fully fleshed out, it depends on how reliable broadcast is going to use